server/src/main/java/org/opensearch/action/search/SearchRequestOperationsListener.java

```diff
@@ -32,6 +40,15 @@ void onRequestStart(SearchRequestContext searchRequestContext) {}
 
     void onRequestEnd(SearchPhaseContext context, SearchRequestContext searchRequestContext) {}
```

Comment:
nit: rename to `searchPhaseContext`

Reply:
this is actually outside of the scope of my change, but yeah it make sense to change the name to make it more clear.